README.md

```diff
@@ -7,21 +7,24 @@ openFrameworks is a C++ toolkit for creative coding.  If you are new to OF, welc
-Windows Visual Studio 32bits | [![Build status](https://appveyor-matrix-badges.herokuapp.com/repos/arturoc/openFrameworks/branch/master/2)](https://ci.appveyor.com/project/arturoc/openFrameworks/branch/master)
-Windows Visual Studio 64bits | [![Build status](https://appveyor-matrix-badges.herokuapp.com/repos/arturoc/openFrameworks/branch/master/3)](https://ci.appveyor.com/project/arturoc/openFrameworks/branch/master)
-Linux 64                     | [![Linux 64 Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="linux64"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Linux armv6l                 | [![Linux armv6l Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="linuxarmv6l"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Linux armv7l                 | [![Linux armv7l Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="linuxarmv7l"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Emscripten                   | [![Emscripten Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="emscripten"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-macos                        | [![macos Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="osx"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-macos makefiles              | [![macos makefiles Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=OPT="makefiles"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-iOS                          | [![iOS Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="ios"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-tvos                         | [![tvos Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=TARGET="tvos"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Android Arm7                 | [![Android Arm7 Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=GRADLE_TARGET="compileArm7DebugSources"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
-Android X86                  | [![Android X86 Build Status](http://badges.herokuapp.com/travis/openframeworks/openFrameworks?env=GRADLE_TARGET="compileX86DebugSources"&label=build&branch=master)](https://travis-ci.org/openframeworks/openFrameworks)
+* **Master** branch has the newest, most recently updated code
```

Comment:
- The **master** branch contains the newest, most recently updated code. This code is packaged and available for download in the "Nightly Builds" section of [openframeworks.cc/download](https://openframeworks.cc/download/).
- The **stable** branch contains the code corresponding to the last stable openFrameworks release. This stable code is packaged and available for download at [openframeworks.cc/download](https://openframeworks.cc/download/).

Reply:
super. thanks. Just added this clarification.